other/require-cpu-limits/require-cpu-limits.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-cpu-limits
+  annotations:
+    policies.kyverno.io/title: Require CPU Limits
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kubernetes-version: "1.26"
+    policies.kyverno.io/description: >-
+      Setting CPU limits on pods ensures fair distribution of resources, preventing any single pod from monopolizing CPU and impacting the performance of other pods. This practice enhances stability, predictability, and cost control, while also mitigating the noisy neighbor problem and facilitating efficient scaling in Kubernetes clusters. This policy ensures that cpu limits are set on every container.
+spec:
+  validationFailureAction: Audit
+  background: true
+  rules:
+  - name: check-cpu-limits
+    match:
+      resources:
```

Comment:
All policies should be using `any`/`all` under the match and exclude conditions.